docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -166,6 +158,11 @@ fields:
   - Link to original form: interview.original_form
     hint: http://masslegalhelp.org/my_form
     required: False
+  - note: |
+      Write a short title that will show up on small screens. Example:[BR]
+      [FILE al_weaver_small_screen_header.png, 100%, Tab text]
```

Comment:
I'm not sure how useful the example here is. The image looks like it could be interacted with at first glance, and it takes a bit of thinking to realize where that would be on interview screens. Prompt is perfect though, I think it'll stand on it's own okay.